pkg/common/label.go

```diff
@@ -29,6 +29,30 @@ const (
 	LabelFluidSchedulingStrategyFlag = LabelAnnotationPrefix + "enable-scheduling-strategy"
 )
 
+// LabelToModify modifies the labelKey in operationType.
+type LabelToModify struct {
+	LabelKey      string
```

Comment:
Do we need LabelKey in uppercase? As you know uppercase  means public attributes of the struct.

Reply:
Yes. In`pkg/utils/label.go` we need to use it. 

```
case common.AddLabel, common.UpdateLabel:
	node.Labels[labelToModify.LabelKey] = labelToModify.LabelValue
```

Reply:
Can we use Get/Set function instead of using attribute directly.

Reply:
Does LabelKey need to be uppercase?